config/webpack.hash.config.js

```diff
@@ -2,14 +2,16 @@
 
 const path = require('path')
 const fs = require('fs')
+const mobile = process.env.MOBILE || false
+const outputFolder = mobile ? 'mobile/www' : 'build'
 
 module.exports = {
   plugins: [
     // Extracts Hash in external file for reference
     function () {
       this.plugin('done', (stats) => {
         fs.writeFileSync(
-          path.join(__dirname, '../build/assets.json'),
+          path.join(__dirname, '../' + outputFolder + '/assets.json'),
```

Comment:
Like in _base_ config, but I ask myself if we shouldn't export those values in a dedicated file, to centralize them. Any preference?